BrowserKit/Sources/ComponentLibrary/BottomSheet/BottomSheetViewController.swift

```diff
@@ -175,7 +175,9 @@ public class BottomSheetViewController: UIViewController,
         view.addSubviews(dimmedBackgroundView, topTapView, sheetView)
         view.accessibilityElements = [closeButton, sheetView]
 
-        contentViewBottomConstraint = sheetView.bottomAnchor.constraint(equalTo: view.bottomAnchor)
+        let contentViewBottomConstraint = sheetView.bottomAnchor.constraint(equalTo: view.bottomAnchor)
+        self.contentViewBottomConstraint = contentViewBottomConstraint
```

Comment:
Is this change needed as part of the unwrapped optional change? Couldn't we continue to set the classes property directly without needing to create an intermediate property?

Reply:
Ah I see. That makes sense!

Reply:
Hi @Cramsden, as suggested by @PARAIPAN9 in #23833 it's actually more consistent with the rest of the codebase to just activate the newly created constraint in place after initialising it as a class property instead of using a temporary local property